cmd/coder/sync.go

```diff
@@ -79,8 +81,10 @@ func (cmd *syncCmd) Run(fl *pflag.FlagSet) {
 		err = s.Run()
 	}
 
-	if err == NoRsync {
+	if fmt.Sprintf("%v", err) == fmt.Sprintf("%v", errIncompatRsync) {
 		flog.Fatal("no compatible rsync present on remote machine")
+	} else if fmt.Sprintf("%v", err) == fmt.Sprintf("%v", errStreamErrRsync) {
+		flog.Fatal("error in rsync protocol datastream (no installed remote rsync?)")
```

Comment:
These changes were for error checking fixed in another branch. Not strictly to do with this review